llama_index/readers/file/base.py

```diff
@@ -84,6 +85,15 @@ def __init__(
 
         self.errors = errors
 
+       # Path validation
+        if input_dir and not os.path.isdir(input_dir):
```

Comment:
while we are validating, it might also be good to check if the input dir has any files?

Reply:
> while we are validating, it might also be good to check if the input dir has any files?

acknowledged.